app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,287 @@
+@import 'bootstrap';
+
+$white: #ffffff;
+$black: #000000;
+$yellow: #feb600;
+$grey: #808080;
+$darkGrey: #a9a9a9;
+$dimGrey: #696969;
+$red: #ff0000;
+$lightYellow: #ffd700;
+
+html, body {
```

Comment:
Are you sure you want to apply these CSS on `html` and `body`? :o what is this for anyways?